src/bidiMapper/modules/network/CollectorsStorage.ts

```diff
@@ -38,6 +41,16 @@ export class CollectorsStorage {
   }
 
   addDataCollector(params: Network.AddDataCollectorParameters) {
+    if (
+      params.maxEncodedDataSize < 1 ||
+      params.maxEncodedDataSize > 200 * 1000 * 1000
```

Comment:
Let's extract into a const and use `200_000_000`

Reply:
done